packages/@adobe/spectrum-css-temp/components/colorarea/skin.css

```diff
@@ -17,10 +14,6 @@
       &:before {
         box-shadow: inset 0 0 0 var(--spectrum-colorarea-border-size) var(--spectrum-colorarea-border-color-disabled);
       }
-
-      .spectrum-ColorArea-gradient {
-        display: none;
-      }
     }
   }
 
```

Comment:
Add `--spectrum-colorarea-border-color: ButtonBorder;` to `forced-colors: active` style declaration at: https://github.com/adobe/react-spectrum/blob/49ba0e7cec1f22f6da62b2a49872169594f661a1/packages/%40adobe/spectrum-css-temp/components/colorarea/skin.css#L21-L23

```css
 .spectrum-ColorArea { 
   --spectrum-colorarea-fill-color-disabled : GrayText; 
   --spectrum-colorarea-border-color: ButtonBorder;
 } 
```